de.dlr.sc.virsat.model.concept.test/resources/expectedOutputFilesForGenerators/AStructuralElementInstanceValidator.java

```diff
@@ -25,11 +25,10 @@
  * 
  * 
  */	
-public abstract class AStructuralElementInstanceValidator implements IStructuralElementInstanceValidator {
+public abstract class AtestConceptValidator implements IStructuralElementInstanceValidator {

```

Comment:
The name of the class does not fit to the file name. This is unusual for java. Here it is a question of the test case, most likely the testcase is still called a StructuralElementInstance… Please check and adjust.

Just in case, the test output can also be generated and does not need to be adjusted by hand.